providers/evercrypt/EverCrypt.AEAD.fsti

```diff
@@ -40,6 +40,9 @@ open EverCrypt.Error
 /// first.
 
 [@CAbstractStruct]
```

Comment:
I think the syntax for defining a set of attributes is as follows:
```
[@@ CAbstractStruct; Comment "Both encryption and decryption require a state that holds the key.
The state may be reused as many times as desired."]
```
@protz can confirm that